packages/babel-parser/test/fixtures/experimental/class-static-block/invalid-arguments/output.json

```diff
@@ -1,9 +1,6 @@
 {
   "type": "File",
   "start":0,"end":52,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":5,"column":1,"index":52}},
-  "errors": [
-    "SyntaxError: 'arguments' is only allowed in functions and class methods. (3:15)"
-  ],
```

Comment:
This is a regression, `arguments` is not allowed directly within static block. We may need to revert pushing `SCOPE_FUNCTION` to static blocks.

Reply:
@JLHwung thanks! i hope now it will run correctly